osu.Game/Overlays/MusicController.cs

```diff
@@ -182,9 +182,14 @@ public bool Play(bool restart = false)
         /// <summary>
         /// Stop playing the current track and pause at the current position.
         /// </summary>
-        public void Stop()
+        /// <param name="requestedByUser">
+        /// Whether the request to stop was issued by the user rather than internally.
+        /// Specifying <c>true</c> will ensure that other methods like <see cref="EnsurePlayingSomething"/>
+        /// will not resume music playback until the next explicit call to <see cref="Play"/>.
+        /// </param>
+        public void Stop(bool requestedByUser = true)
```

Comment:
I think I'd prefer the default to be false, personally. Also maybe we want a matching parameter on `Play`..

@smoogipoo thoughts?

Reply:
`false` probably makes more sense in most usages, so I'd agree.